examples/shader/compute_shader_game_of_life.rs

```diff
@@ -19,8 +19,10 @@ fn main() {
     App::new()
         .insert_resource(ClearColor(Color::BLACK))
         .insert_resource(WindowDescriptor {
-            // uncomment for unthrottled FPS
-            // vsync: false,
+            // uncomment one for unthrottled FPS
+            // if it doesn't work try the other one
+            //present_mode: PresentMode::Mailbox,
+            //present_mode: PresentMode::Immediate,
```

Comment:
```suggestion
            // present_mode: PresentMode::Mailbox,
            // present_mode: PresentMode::Immediate,
```
Could you explain more the difference than "just try the other one"?

Reply:
I would have already, but I thought it might be better to keep it brief. Will do!